src/common/hooks/useDraggableHandle.tsx

```diff
@@ -111,7 +111,7 @@ export function useDraggableHandle(
   }, [dragState.dragging]);
 
   useEffect(() => {
-    const handler = (e) => dispatch(e as DragActions);
+    const handler = (e: any) => dispatch(e as DragActions);
```

Comment:
The correct type here is `MouseEvent` (for future reference, when the type is not known, we should use the `unknown` type instead of `any`)